packages/manager/src/features/IAM/Users/UsersTable/UserRow.tsx

```diff
@@ -0,0 +1,99 @@
+import { useTheme } from '@mui/material/styles';
+import React from 'react';
+
+import { Avatar } from 'src/components/Avatar/Avatar';
+import { DateTimeDisplay } from 'src/components/DateTimeDisplay';
+import { Hidden } from 'src/components/Hidden';
+import { StatusIcon } from 'src/components/StatusIcon/StatusIcon';
+import { TableCell } from 'src/components/TableCell';
+import { TableRow } from 'src/components/TableRow';
+import { useProfile } from 'src/queries/profile/profile';
+import { capitalize } from 'src/utilities/capitalize';
+
+import { UsersActionMenu } from './UsersActionMenu';
+
+import type { User } from '@linode/api-v4';
+import { Link } from 'react-router-dom';
+import { Box, Chip, Stack, Typography } from '@linode/ui';
+
+interface Props {
+  onDelete: (username: string) => void;
+  user: User;
+}
+
+export const UserRow = ({ onDelete, user }: Props) => {
+  const theme = useTheme();
+
+  const { data: profile } = useProfile();
+
+  const isProxyUser = Boolean(user.user_type === 'proxy');
+
+  return (
+    <TableRow data-qa-table-row={user.username} key={user.username}>
+      <TableCell>
+        <Stack alignItems="center" direction="row" spacing={1.5}>
+          <Avatar
+            color={
+              user.username !== profile?.username
+                ? theme.palette.primary.dark
+                : undefined
+            }
+            username={user.username}
+          />
+          <Typography>
+            <Link to={`/iam/users/${user.username}/details`}>
+              {user.username}
+            </Link>
+          </Typography>
+          <Box display="flex" flexGrow={1} />
+          {user.tfa_enabled && <Chip color="success" label="2FA" />}
+        </Stack>
+      </TableCell>
+      <Hidden smDown>
+        <TableCell>{user.email}</TableCell>
```

Comment:
Same comment as above. 

```suggestion
        <TableCell>
          <MaskableText isToggleable text={user.email} />
        </TableCell>
```